core/templates/components/skill-mastery/skill-mastery.component.spec.ts

```diff
@@ -0,0 +1,103 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Unit tests for SkillMasteryViewerComponent
+ */
+
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { SkillMasteryViewerComponent } from './skill-mastery.component';
+import { SkillMasteryListConstants } from
+  'components/skills-mastery-list/skills-mastery-list.constants';
```

Comment:
Keep this on the same line as above... Same below

Reply:
Done